src/main/resources/org/jenkinsci/plugins/docker/traceability/DockerTraceabilityPluginConfiguration/help-createImageFingerprints.html

```diff
@@ -0,0 +1,14 @@
+<div>
+  By default, the plugin registers traceability reports for images, which have
+  fingerprints registered by Jenkins plugins using the functionality from 
+  <a href="https://wiki.jenkins-ci.org/display/JENKINS/Docker+Commons+Plugin">
+    Docker Commons Plugin</a>. 
+    This behavior may may be an obstacle if a you want to trace fingerprints
+  for images created outside Jenkins or by plugins, which have no integration with
```

Comment:
Ditto here. As written, the sentence could be paraphrased

> This behavior may be an obstacle if you want to trace fingerprints for images created outside Jenkins (or by plugins rather than core). These images have no integration with Docker Commons.